jormungandr-lib/src/interfaces/vote.rs

```diff
@@ -157,7 +157,13 @@ pub struct VotePlan {
     #[serde(with = "serde_proposals")]
     proposals: Proposals,
     #[serde(with = "serde_committee_member_public_keys", default = "Vec::new")]
-    committee_member_public_keys: Vec<chain_vote::MemberPublicKey>,
+    pub committee_member_public_keys: Vec<chain_vote::MemberPublicKey>,
+}
+
+#[derive(Clone, Debug, Deserialize, Eq, Hash, PartialEq, Serialize)]
+pub struct VotePlanKey {
```

Comment:
If this is a test-only interface maybe we can put the definition in jormungandr-testing-utils? I don't remember if we have a convention for this, but IMO it would make reading the blockchain interface a bit harder as it's "polluted" with testing stuff

Reply:
Good point, I moved it to `jormungandr-testing-utils`